qute.ls/com.redhat.qute.ls/src/main/resources/com/redhat/qute/project/datamodel/qute-resolvers.json

```diff
@@ -25,6 +25,23 @@
 				"{#for r in recordsList.take(3)}"
 			],
 			"url": "https://quarkus.io/guides/qute-reference#collections"
+		},
+		{
+			"signature": "or(self: T, arg : java.lang.Object) : boolean",

```

Comment:
or doesn't return a boolean but a base object or a default value. Today it's not possible to returnthe union of base object type and default value type, Let's return the base object type. To do that the signature must be :

`"signature": "or(base : T, arg : java.lang.Object) : T"`

Reply:
Please write test like https://github.com/redhat-developer/quarkus-ls/blob/master/qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/resolvers/OrEmptyValueResolverTest.java